state/model.go

```diff
@@ -1040,16 +1046,22 @@ func (m *Model) checkEmpty() error {
 		}
 		return errors.Annotatef(err, "getting entity references for model %s", m.UUID())
 	}
+	// These errors could be potentially swallowed as we re-try to destroy model.
+	// Let's, at least, log them for observations.
```

Comment:
Nice :) With my English, both plural and singular work here but I appreciate your feedback! Changed :)